docs/articles/documentation/using-testcafe/common-concepts/browser-support.md

```diff
@@ -5,57 +5,71 @@ permalink: /documentation/using-testcafe/common-concepts/browser-support.html
+## Remote Browsers
 
-You can install browser provider packages from npm as you would install any other plugin. See [Installing Plugins](../../extending-testcafe/index.md#installing-plugins).
+TestCafe can run tests in browsers on remote devices. The only requirement is a device should have network access to the TestCafe server.
 
-## Specifying Browsers for Test Task
+At first, you need to create a remote browser connection. You can do this from CLI by specifying the `remote` *alias* or from API by using the [createBrowserConnection](../programming-interface/testcafe.html#createbrowserconnection) method.
 
-If you are working with TestCafe through the command line,
-[list the desired browsers](../command-line-interface.md#browser-list)
-within the `testcafe` command.
+After that, TestCafe will provide URLs that you need to open in required browsers on your remote device to start testing.
 
-In case you are using the API, specify the browsers
-with the [runner.browsers](../programming-interface/runner.md#browsers) method.
+## Cloud Browsers
+
+TestCafe allows you to use cloud browsers. These browsers can be accessed through *browser provider plugins*. For example, there is the [testcafe-browser-provider-saucelabs](https://www.npmjs.com/package/testcafe-browser-provider-saucelabs) plugin for testing in [Sauce Labs](https://saucelabs.com/).
+
+To specify a Sauce Labs browser, use a browser alias that consists of the `saucelabs` prefix and the name of the browser itself, for example, `saucelabs:Chrome@52.0:Windows 8.1`.
+To obtain all the available aliases for the Sauce Labs browser provider, run the `testcafe --list-browsers saucelabs` command.
+
+If you need to use another cloud browser, see [Other Browsers](#other-browsers).
+
+## Other Browsers
+
+To use a different browser, create a custom [browser provider plugin](../../extending-testcafe/browser-provider-plugin/).
+
+Created browser provider packages can be published to npm. The provider package name consists of two parts - the `testcafe-browser-provider-` prefix and the name of a provider itself; e.g., `testcafe-browser-provider-saucelabs`.
+So, you can search for available browser provider packages on npm by the `testcafe-browser-provider-` prefix: [https://www.npmjs.com/search?q=testcafe-browser-provider](https://www.npmjs.com/search?q=testcafe-browser-provider).
+
+For example, there is the [testcafe-browser-provider-phantomjs](https://www.npmjs.com/package/testcafe-browser-provider-phantomjs) plugin for tesing in the [PhantomJS](http://phantomjs.org/) headless browser.
 
-To identify browsers, different approaches are used.
+When running tests, you can specify a browser accessed through a provider plugin by using a *browser alias*.
+The alias consists of the browser provider name and the name of the browser itself (the latter can be omitted); for example, `saucelabs:Chrome@52.0:Windows 8.1` or `phantomjs`.
```

Comment:
Don't leak another package documentation here. It's enough to share a link to recipe here.